### ISSUE TITLE
Wrong command to launch the app?

### ISSUE THREAD

**Issue by alinnert** [fuse-box/fuse-box-electron-seed#1]
I'm not sure if other platforms behave differently but on Windows the app won't launch after running `npm start` in the console. But I already found why:

Line 31 in `fuse.js` says the following:

~~~javascript
spawn(`${__dirname}/node_modules/electron/cli.js`, [__dirname])
~~~

It works if I change it to:

~~~javascript
spawn('node', [`${ __dirname }/node_modules/electron/cli.js`,  __dirname ])
~~~

Basically running `node .../cli.js dist` instead of just `.../cli.js dist`.

**Comment by devmondo** [comment]
Fixed now, thanks a lot for the support :)
